### DISCUSSION TITLE
UseFormMethods is missing?

### DISCUSSION THREAD

**Discussion by founderpad** [react-hook-form/react-hook-form#7909] [Q&A]
I'm trying to build a generic form to cut down on a lot of boilerplate but some of the docs aren't referencing the newer types. E.g.,

```
UseFormMethods  -> UseFormReturn
UseFormOptions -> UseFormProps
```

Should probably be updated. I've seen it in a few places

**Comment by bluebill1049** [comment]
It's on the TS page

https://react-hook-form.com/ts#UseFormProps
https://react-hook-form.com/ts#UseFormReturn
